tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -318,6 +320,231 @@ function ( $passed_order ) use ( $order ) {
+		$this->mock_webhook_handler->expects( $this->once() )
+			->method( 'process_refund' )
+			->with( $order->get_id(), $order->get_total(), $this->isType( 'string' ) )
+			->willReturn( true );
+
+		$this->mock_webhook_handler->process_deferred_webhook( 'payment_intent.succeeded', $data, $notification );
+
+		$updated_order = wc_get_order( $order->get_id() );
+		$this->assertEquals( OrderStatus::CANCELLED, $updated_order->get_status() );
+		$this->assertEquals( 'yes', $updated_order->get_meta( self::META_REFUNDED_AFTER_CANCELLATION ) );
+		$this->assertOrderHasNoteContaining( $updated_order, 'automatically refunded' );
+	}
```

PHPMD (2.15.0)

[error] The method test_cancelled_order_captured_payment_is_refunded is not named in camelCase. (undefined)

(CamelCaseMethodName)